Source/diablo.cpp

```diff
@@ -8,6 +8,7 @@
 #include "../3rdParty/Storm/Source/storm.h"
 #include "../DiabloUI/diabloui.h"
 #include <config.h>
+#include "misc\config.h" //For reading options from config during startup

```

Comment:
Probably this will help:
```suggestion
#include "misc/config.h" //For reading options from config during startup

```